### PR TITLE
adding getFold as a separate function and adding sampleAClass. Correct...

### PR DESCRIPTION
**Pull request by jwist**
…ting minor issue with size of folds when length dataset is not a manifold of k

### CODE REVIEW

**Review by stropitek — COMMENTED** [review]
Why are committing the dist and lib folders? Other than that LGTM.

For most packages in mljs it's enough to run the rollup script during the prepublish step and to let git ignore the lib folder.

File: package.json [hunk 1]
```diff
@@ -48,11 +49,16 @@
...
     "jest": "^25.1.0",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "rollup": "^1.30.1",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0"
   },
   "homepage": "https://github.com/mljs/cross-validation#readme",
   "dependencies": {
+    "jest-matcher-deep-close-to": "^1.3.0",
```
Comment (targos) [inline package.json]: this should be in devDependencies

File: package.json [hunk 2]
```diff
@@ -48,11 +49,16 @@
...
     "ml-combinations": "^1.1.0",
-    "ml-confusion-matrix": "^0.4.0"
+    "ml-confusion-matrix": "^0.4.0",
+    "ml-dataset-iris": "^1.1.1"
```
Comment (targos) [inline package.json]: this should be in devDependencies

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export default {
+  input: 'src/index.js',
+  output: {
+    format: 'cjs',
+    file: 'lib/index.js',
+  },
+  external: [
+    'jest-matcher-deep-close-to',
```
Comment (targos) [inline rollup.config.js]:
```suggestion
```

File: rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    'ml-combinations',
+    'ml-dataset-iris',
```
Comment (targos) [inline rollup.config.js]:
```suggestion
```

**Comment by jwist** [comment]
> Why are committing the dist and lib folders? Other than that LGTM.
> 
> For most packages in mljs it's enough to run the rollup script during the prepublish step and to let git ignore the lib folder.

this is clearly my mistake.

File: package.json (hunk 1 above)
Comment (jwist) [reply package.json]: yes, sorry for that.

File: package.json (hunk 2 above)
Comment (jwist) [reply package.json]: I moved data iris, the confusion matrix should be used by the prediction method, although it is not yet supported.

File: rollup.config.js (hunk 3 above)
Comment (jwist) [reply rollup.config.js]: done

File: rollup.config.js (hunk 4 above)
Comment (jwist) [reply rollup.config.js]: done

**Review by stropitek — APPROVED** [review]

**Pull request merged**